src/components/ClusterManager.js

```diff
@@ -363,6 +365,22 @@ export default ajaxCaller(class ClusterManager extends PureComponent {
     this.resetUpdateInterval()
   }
 
+  componentDidUpdate(prevProps) {
+    const prevCluster = _.last(_.sortBy('createdDate', _.remove({ status: 'Deleting' }, prevProps.clusters))) || {}
+    const { status, jupyterUserScriptUri } = this.getCurrentCluster() || {}
+    if (status === 'Error' && prevCluster.status !== 'Error') {
+      notify('error', 'Error creating cluster', {
+        message: h(Fragment, [
+          strong(['Failed action: ']), jupyterUserScriptUri,
+          h(Clickable, {
+            onClick: () => this.setState({ errorModalOpen: true }),
```

Comment:
Since these notifications can persist across pages, the original cluster manager might have unmounted when this is clicked. We could move the state and modal into the notification content itself.